File: 백준/Bronze/25304. 영수증/영수증.cc
#include <iostream>
using namespace std;
int main(void)
{
    int money,count,money1,count1,i;
    int allmoney=0;
    cin>>money;
    cin>>count;
    for(i=0;i<count;i++)
    {
        cin>>money1>>count1;
        allmoney+=(money1*count1);
    }
    if(allmoney==money)
    {
        cout<<"Yes"<<endl;
    }
    else
    {
        cout<<"No"<<endl;
    }
}

File: 백준/Bronze/25304. 영수증/영수증.java
import java.util.Scanner;
public class Main{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int allmoney=0;
        int money=sc.nextInt();
        int count=sc.nextInt();
        for(int i=0;i<count;i++){
            int money1=sc.nextInt();
            int count1=sc.nextInt();
            allmoney+=(money1*count1);
        }
        if(allmoney==money)
        {
            System.out.println("Yes");
        }
        else
        {
            System.out.println("No");
        }
    }
}